clusterloader2/pkg/config/cluster.go

```diff
@@ -40,6 +40,8 @@ type ClusterConfig struct {
 	MasterName                 string
 	KubemarkRootKubeConfigPath string
 	DeleteStaleNamespaces      bool
+	IsSSHSupported             bool
+	IsPprofSupported           bool
```

Comment:
Similarly to the ssh we should make it clear what it means. Could you either change the name to make it more self documenting or add a comment (or both)?

Reply:
how's that?